libcalico-go/lib/backend/k8s/conversion/constants.go

```diff
@@ -18,6 +18,7 @@ const (
 	NamespaceLabelPrefix            = "pcns."
 	NamespaceProfileNamePrefix      = "kns."
 	K8sNetworkPolicyNamePrefix      = "knp.default."
+	K8sAdminNetworkPolicyNamePrefix = "kanp.anp."
```

Comment:
This prefix represents `Kubernetes AdminNetworkPolicy` in `AdminNetworkPolicy` tier. But maybe the second `anp` is redundant, and can be removed.